### ISSUE TITLE
Monitoring

### ISSUE THREAD

**Issue by sabman** [datmo/datmo#295]
Thanks for this project. Curious to know why you got rid of the Monitoring part of the lib.

**Comment by shabazpatel** [comment]
The monitoring part of it exists on CLI. Would you be interested in having an SDK for monitoring?

**Comment by sabman** [comment]
Yes possibly if we wanna add it to a worker. At the time I was looking for a solution to a specific problem. Not as urgent now.
I was also just curious coz of this commit: https://github.com/datmo/datmo/commit/17bded54d9392c217de6c47bb4519e3018f951c7
